src/engine_gui.cpp

```diff
@@ -325,11 +325,10 @@ void DrawVehicleEngine(int left, int right, int preferred_x, int y, EngineID eng
  */
 void EngList_Sort(GUIEngineList *el, EngList_SortTypeFunction compare)
 {
-	size_t size = el->size();
 	/* out-of-bounds access at the next line for size == 0 (even with operator[] at some systems)
```

Comment:
is this true any more?

Reply:
I don't know, but maybe the comment can go, as anyway sorting less than 2 elements is useless.